add_docstrings.py

```diff
@@ -155,31 +311,62 @@ def leave_ClassDef(
             )
 
     def object_fullname(self, final_part: str) -> str:
+        """Given an unqualified name, convert it to a fully qualified name.
+
+        For example, given the name `method`, if we are visiting a class `Foo`
+        inside a class `Bar` inside a module `spam`, this will return `spam.Bar.Foo.method`.
+
+        If the name is mangled, it will be mangled according to the current class
+        namespace: if we were given the name `__method`, we would instead return
+        `spam.Bar.Foo._Foo__method`.
```

Comment:
```suggestion
        If the name starts with `__`, it will be mangled according to the current class
        namespace: if we were given the name `__method`, we would instead return
        `spam.Bar.Foo._Foo__method`.
```